### PR TITLE
New blog on Fair-Ease and Eurosciencegateway

### PR DESCRIPTION
**Pull request by Marie59**
Feedback from the galaxy training session held in Lille with @bgruening. Is it good for you ?

### CODE REVIEW

**Comment by Marie59** [comment]
Thank you @yvanlebras !!

**Comment by yvanlebras** [comment]
Hi everyone, THANK YOU Marie! Duno why, I was not able to create a new branch to propose a PR on your index file... and testing direct commit... works :( Sorry. Please, don't hesitate to change the proposed modification. In particular, I was adding this "after a bief introduction from the Galaxy for Ecology initiative" sentence, but duno if it is relevant or not. Don't hesitate to revert / change !

**Comment by Marie59** [comment]
Everything seems good to me ! I think it's relevant it was part of the Galaxy discovery !

File: content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Stronger together: FAIR-EASE and EuroScienceGateway join forces" 
+tease: "Seconde blog post presenting the outcome of a galaxy training session"
+authors: "Marie Jossé"
+source_blog: "https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces"
```
Comment (bgruening) [inline content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md:5]: I need to check, but can it be that this tag will simply redirect the user to this page?

File: content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Stronger together: FAIR-EASE and EuroScienceGateway join forces" 
+tease: "Seconde blog post presenting the outcome of a galaxy training session"
```
Comment (bgruening) [inline content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md:3]:
```suggestion
tease: "Presenting some outcomes from the first joined FAIR-EASE / EuroScienceGateway workshop"
```

**Review by bgruening — COMMENTED** [review]
Ah I see, sorry, that is super confusing. There are blogs and news.
I think what we want is always News ... not sure for what are blogs good for.

Can you please create a simple news item instead?

Sorry for the confusion.

File: content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md (hunk 1 above)
Comment (Marie59) [reply content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md:5]: Uh perhaps I thought it would just write down the link in the description

**Comment by Marie59** [comment]
Should we now close this PR ? 
Thanks @bgruening

**Pull request closed without merging**